src/main/resources/db/changelog/changelog-master.xml

```diff
@@ -9,5 +9,6 @@
     <property name="uuid_function" value="uuid_generate_v4()" dbms="postgresql"/>
 
     <includeAll path="1.0.0" relativeToChangelogFile="true"/>
+    <includeAll path="1.0.1" relativeToChangelogFile="true"/>
```

Comment:
Ich würde die Datei noch in den 1.0.0 Ordner tun, ist ja alles für den ersten Release, oder? Thema Versionierung sollten wir uns sowieso in der größeren Gruppe nochmal anschauen :)